dbt_server/logging.py

```diff
@@ -41,42 +41,55 @@ def add_fields(self, log_record, record, message_dict):
             log_record["workspaceID"] = WORKSPACE_ID
 
 
-# setup json logging for stdout and datadog
-logger = logging.getLogger()
-logger.setLevel(logging.DEBUG)
-stdout = logging.StreamHandler()
-if os.environ.get("APPLICATION_ENVIRONMENT") in ("dev", None):
-    formatter = logging.Formatter(
-        "%(asctime)s - [%(process)d] %(name)s - %(levelname)s - %(message)s"
-    )
-else:
-    formatter = CustomJsonFormatter(
-        "%(timestamp)f %(filename)s %(funcName)s %(levelname)s "
-        "%(lineno)d %(message)s %(module)s %(pathname)s %(process)d "
-        "%(processName)s %(thread)s %(threadName)s %(name)s "
-        "[dd.service=%(dd.service)s dd.env=%(dd.env)s dd.version=%(dd.version)s "
-        "dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s]"
-    )
-stdout.setFormatter(formatter)
-logger.addHandler(stdout)
+def get_log_formatter():
+    if os.environ.get("APPLICATION_ENVIRONMENT") in ("dev", None):
+        formatter = logging.Formatter(
+            "%(asctime)s - [%(process)d] %(name)s - %(levelname)s - %(message)s"
+        )
+    else:
+        formatter = CustomJsonFormatter(
+            "%(timestamp)f %(filename)s %(funcName)s %(levelname)s "
+            "%(lineno)d %(message)s %(module)s %(pathname)s %(process)d "
+            "%(processName)s %(thread)s %(threadName)s %(name)s "
+            "[dd.service=%(dd.service)s dd.env=%(dd.env)s dd.version=%(dd.version)s "
+            "dd.trace_id=%(dd.trace_id)s dd.span_id=%(dd.span_id)s]"
+        )
+    return formatter
+
 
 # Use standard python logger for all dbt-server logs-- these will be sent to
 # stdout but will not be written to task log files
 DBT_SERVER_LOGGER = logging.getLogger("dbt-server")
 DBT_SERVER_LOGGER.setLevel(logging.DEBUG)
+stdout = logging.StreamHandler()
+formatter = get_log_formatter()
+stdout.setFormatter(formatter)
+DBT_SERVER_LOGGER.addHandler(stdout)
 
 # make sure uvicorn is deferring to the root
 # logger to format logs
 logger_instance = logging.root.manager.loggerDict.get("uvicorn")
 if logger_instance:
-    logger.propagate = True
+    logger_instance.propagate = True
     logger_instance.handlers = []
 logger_instance = logging.root.manager.loggerDict.get("uvicorn.error")
 if logger_instance:
-    logger.propagate = True
+    logger_instance.propagate = True
     logger_instance.handlers = []
 
 
+def get_configured_celery_logger():
+    logger = logging.getLogger(__name__)
+    logger.handlers = []
+    formatter = get_log_formatter()
+    file_handler = logging.FileHandler(filename=CELERY_LOG_FILE.get())
+    file_handler.setFormatter(formatter)
+    logger.addHandler(file_handler)
+    logger.setLevel(logging.INFO)
```

Comment:
does this level get propagates down to dbt somehow?

Reply:
No, this level should not affect the dbt level. We should still be getting debug logs from dbt